src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="form__big"
+      action="#"
+      method="post"
+      data-qa="big"
+      >
+      <label>
+        <img
+          class="img-big"
+          src="./images/Search.svg"
+          alt="find">
+
+        <input
+          class="search__big"
+          type="search"
+          name="search_big"
```

Comment:
6 point in checklist
```suggestion
          name="searchBig"
```